### ISSUE TITLE
Support for Mercurial

### ISSUE THREAD

**Issue by Afulton11** [ben01189998819991197253/contrail#16]
The place I work at uses Mercurial instead of git, maybe we could add support for it as well? 
module_for_git() would change to something like module_for_vcs()

**Comment by bepzi** [comment]
Yes, definitely we should do this. I'm still smoothing out some rough edges with the error handling in the 0.2 rewrite, and also I'm encountering some performance issues that need sorted out, but once those are fixed, supporting other VCS's is something I'd like to add.
